lib/connectHapiServerToRedis.js

```diff
@@ -63,23 +63,26 @@ function connectHapiServerToRedis (server, cb) {
       }
     });
   });
+  // FIXIT - remove this once redis is integrated
+  cb(null, {status: "connected, but redis is not built out yet!"}); 
```

Comment:
Added callback so that things won't hang!  Node doesn't like that :stuck_out_tongue_winking_eye: